### ISSUE TITLE
Windows installer complicated choices

### ISSUE THREAD

**Issue by JohnVeness** [owncloud/client#1702]
The Windows installer asks the user to choose between four different types of install: Standard, Full, Minimal or Custom, which affect which icons (Start menu, Desktop, or Quick Launch toolbar) are created. The experience is quite confusing and over the top for what is being selected.
![ownCloud installer](https://cloud.githubusercontent.com/assets/2512915/2758894/6ceeddbc-c997-11e3-88f2-e1b1f447c518.png)

I suggest instead of the above described screen, there should be a screen just with tickboxes for which icons should be selected, as the installers for KeePass and LibreOffice do.
![KeePass installer](http://northamptoncomputerrepair.com/Portals/166626/images/KeePass%201.png)
![LibreOffice installer](https://www.libreoffice.org/assets/Uploads/Shared-Resources/635readytoinstall.png)

Having said that, I think you can simplify even further. There should always be a Start menu icon created without giving the user a choice (see http://msdn.microsoft.com/en-us/library/windows/desktop/aa511447.aspx). The Quick Launch toolbar is depricated since Windows Vista, the modern equivalent being pinning to the taskbar (which shouldn't be something the installer does). And the Desktop is no longer seen as such an important place to put icons. The user could always copy the icon there themselves if they wanted.

In other words, I think you can skip entirely given the user any choice as to where to put icons, removing an entire screen from the wizard. But if you really think you need to give the user a choice, I suggest a KeePass or LibreOffice-style UI, rather than the Standard/Full/Minimal/Custom treeview component chooser.

**Comment by ogoffart** [comment]
Closing old issue, as we move to .MSI
